### PR TITLE
Move cloud ops to new file_operations module

### PR DESCRIPTION
**Pull request by EmmanuelMess**
## PR Info

Start move to another module for file operations

#### Issue tracker   
Fixes will automatically close the related issue

Fixes #
-or-   
Addresses #

#### Release  
Addresses release/
  
#### Test cases
- [ ] Covered
  
#### Manual testing
- [ ] Done  
  
If yes,  
- Device:
- OS:

#### Build tasks success  
Successfully running following tasks on local 
- [x] `./gradlew assembledebug`
- [x] `./gradlew spotlessCheck`

#### Related PR  
Related to PR #

#### Additional Info

### CODE REVIEW

**Comment by EmmanuelMess** [comment]
@VishalNehra Codacy failed with no errors, what?

**Review by TranceLove — APPROVED** [review]
Should be good to go, though it's best if the relevant test cases can be moved to file_operations module too (costly however, may cause cyclic dependency).

**Comment by EmmanuelMess** [comment]
>  though it's best if the relevant test cases can be moved to file_operations module too (costly however, may cause cyclic dependency).

This will be done as dependencies are moved.

File: file_operations/build.gradle [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+            returnDefaultValues = true
+        }
+    }
+}
+
+ext {
+    robolectricVersion = '4.3.1'
```
Comment (VishalNehra) [inline file_operations/build.gradle:38]: Redundant version declarations

**Review by VishalNehra — CHANGES_REQUESTED** [review]

File: file_operations/build.gradle (hunk 1 above)
Comment (EmmanuelMess) [reply file_operations/build.gradle:38]: Yes, the idea is to leave them redundant, so that app and file_operations are more independent.

**Review by VishalNehra — APPROVED** [review]

**Pull request merged**